packages/cliqz-component-ui-movie-showtimes/src/index.tsx

```diff
@@ -0,0 +1,337 @@
+      color: '#757575',
+      fontSize: 13.5,
+    },
+    cinemaContainer: {
+      alignItems: 'center',
+      borderBottomWidth: 0.5,
+      borderColor: 'rgba(0, 0, 0, 0.04)',
+      flexDirection: 'row',
+      flexWrap: 'wrap',
+      maxWidth: 648,
+      overflow: 'hidden',
+      paddingBottom: 16,
+      paddingTop: 16,
+    },
+    cinemaHeaderContainer: {
+      flex: 2,
+      marginRight: isSmallScreen ? 0 : 4,
+      minWidth: isSmallScreen ? 300 : 268,
+    },
+    cinemaNameText: {
+      color: '#212121',
+      fontSize: 16,
+    },
+    cityContainer: {
+      backgroundColor: '#EEEEEE', // TODO: ask alexei
+    },
+    cityLocationContainer: {
+      height: 28,
+      justifyContent: 'center',
+      marginTop: 6,
+      paddingLeft: 5,
+      paddingRight: 5,
+      width: 147.5,
+    },
+    cityText: {
+      color: '#616161',
+      fontSize: 13.5,
+    },
+    container: {
+      marginLeft: margin,
+      marginRight: margin,
+      paddingTop: 25,
+      width: width - 2 * margin,
+    },
+    locationContainer: {
+      alignItems: 'center',
+      borderColor: '#2F7D32', // TODO: ask alexei
+      borderWidth: 1,
+      marginLeft: 5,
+    },
+    locationText: {
+      color: '#2F7D32',
+      fontSize: 13.5,
+    },
+    moreLessButton: {
+      alignItems: isSmallScreen ? 'center' : 'flex-start',
+      height: 51,
+      paddingBottom: 16,
+      paddingTop: 16,
+    },
+    moreLessButtonText: {
+      color: '#757575',
+      fontSize: 14,
+      fontWeight: '500',
+      textTransform: 'uppercase',
+    },
+    movieTitle: {
+      color: '#212121',
+      fontSize: 19.5,
+      marginTop: 16,
+    },
+    movieTitleContainer: {
+      height: 64,
+      justifyContent: 'center',
+    },
+    showContainer: {
+      alignItems: 'center',
+      backgroundColor: '#f4f3fa', // TODO: ask Alexei
+      borderRadius: 4,
+      flexDirection: 'row',
+      height: 26,
+      marginBottom: 5,
+      marginRight: 10,
+      marginTop: 5,
+      padding: 4,
+    },
+    showTimesContainer: {
+      alignItems: 'center',
+      flex: 1,
+      flexDirection: 'row',
+      flexWrap: 'wrap',
+      minWidth: isSmallScreen ? 150 : 134,
+    },
+    showTimesExtraText: {
+      color: '#311B92',
+      fontSize: 9,
+      lineHeight: 10,
+    },
+    showTimesSeparator: {
+      backgroundColor: 'white',
+      height: 26,
+      marginLeft: 4,
+      marginRight: 4,
+      width: 0.5,
+    },
+    showTimesTime: {
+      color: '#311B92',
+      fontSize: 14,
+      lineHeight: 20,
+    },
+    tableHeadersContainer: {
+      flexDirection: 'row',
+      height: 32,
+      overflow: 'hidden',
+    },
+    title: {
+      color: '#757575',
+      fontSize: 19.5,
+      fontWeight: '400',
+    },
+  });
+};
+
+export class MovieShowtimes extends React.PureComponent<
+  MovieProps,
+  MovieState
+> {
+  public readonly state: MovieState = {
+    day: 0,
+    limit: true,
+    width: Dimensions.get('window').width,
```

Comment:
are we sure this is being evaluated when the class is instantiated? shouldn't it be called when componentWillMount?